### ISSUE TITLE
python export.py --weights yolov5s.pt --include tfjs

### ISSUE THREAD

**Issue by gamefake588** [ultralytics/yolov5#5436]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

 export failure: module 'tensorflow.tools.docs.doc_controls' has no attribute 'inheritable_header'

### Additional

Package                 Version
----------------------- ---------------------
absl-py                 0.15.0
astunparse              1.6.3
cachetools              4.2.4
certifi                 2021.10.8
charset-normalizer      2.0.7
clang                   5.0
colorama                0.4.4
coremltools             5.0
cycler                  0.11.0
flatbuffers             1.12
gast                    0.4.0
google-auth             2.3.2
google-auth-oauthlib    0.4.6
google-pasta            0.2.0
grpcio                  1.41.1
h5py                    3.1.0
idna                    3.3
keras                   2.6.0
Keras-Preprocessing     1.1.2
kiwisolver              1.3.2
Markdown                3.3.4
matplotlib              3.4.3
mpmath                  1.2.1
numpy                   1.19.5
oauthlib                3.1.1
onnx                    1.10.2
onnx-simplifier         0.3.6
onnxoptimizer           0.2.6
onnxruntime             1.9.0
opencv-python           4.5.4.58
opt-einsum              3.3.0
packaging               21.2
pandas                  1.3.4
Pillow                  8.4.0
pip                     21.2.2
protobuf                3.19.1
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pyparsing               2.4.7
python-dateutil         2.8.2
pytz                    2021.3
PyYAML                  6.0
requests                2.26.0
requests-oauthlib       1.3.0
rsa                     4.7.2
scikit-learn            0.19.2
scipy                   1.4.1
seaborn                 0.11.2
setuptools              58.0.4
six                     1.15.0
sympy                   1.9
tensorboard             2.7.0
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.0
tensorflow              2.6.0
tensorflow-estimator    2.7.0
tensorflow-hub          0.12.0
tensorflowjs            3.11.0
termcolor               1.1.0
thop                    0.0.31.post2005241907
torch                   1.10.0
torchvision             0.11.1
tqdm                    4.62.3
typing-extensions       3.7.4.3
urllib3                 1.26.7
Werkzeug                2.0.2
wheel                   0.37.0
wincertstore            0.2
wrapt                   1.12.1

**Comment by gamefake588** [comment]
export: data=data\coco128.yaml, weights=weights/yolov5s.pt, imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=False, simplify=False
, opset=13, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['tfjs']
YOLOv5  2021-10-25 torch 1.10.0+cpu CPU

Fusing layers...
F:\code-tool\Anaconda\bin\envs\new_yolov5\lib\site-packages\torch\functional.py:445: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered i
nternally at  ..\aten\src\ATen\native\TensorShape.cpp:2157.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Model Summary: 213 layers, 7225885 parameters, 0 gradients

PyTorch: starting from weights\yolov5s.pt (14.7 MB)

TensorFlow saved_model: starting export with tensorflow 2.6.0...

                 from  n    params  module                                  arguments
2021-11-01 23:22:38.001901: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2021-11-01 23:22:38.014640: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: DESKTOP-75E31UM
2021-11-01 23:22:38.022990: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: DESKTOP-75E31UM
2021-11-01 23:22:38.031158: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructio
ns in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
  0                -1  1      3520  models.common.Conv                      [3, 32, 6, 2, 2]
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]
  2                -1  1     18816  models.common.C3                        [64, 64, 1]
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]
  4                -1  1    115712  models.common.C3                        [128, 128, 2]
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]
  6                -1  1    625152  models.common.C3                        [256, 256, 3]
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]
  8                -1  1   1182720  models.common.C3                        [512, 512, 1]
  9                -1  1    656896  models.common.SPPF                      [512, 512, 5]
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 12           [-1, 6]  1         0  models.common.Concat                    [1]
 13                -1  1    361984  models.common.C3                        [512, 256, 1, False]
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 16           [-1, 4]  1         0  models.common.Concat                    [1]
 17                -1  1     90880  models.common.C3                        [256, 128, 1, False]
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]
 19          [-1, 14]  1         0  models.common.Concat                    [1]
 20                -1  1    296448  models.common.C3                        [256, 256, 1, False]
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]
 22          [-1, 10]  1         0  models.common.Concat                    [1]
 23                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 24      [17, 20, 23]  1    229245  models.yolo.Detect                      [80, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512], [640, 640]]
Model: "model"
__________________________________________________________________________________________________
Layer (type)                    Output Shape         Param #     Connected to
==================================================================================================
input_1 (InputLayer)            [(1, 640, 640, 3)]   0
__________________________________________________________________________________________________
tf_conv (TFConv)                (1, 320, 320, 32)    3488        input_1[0][0]
__________________________________________________________________________________________________
tf_conv_1 (TFConv)              (1, 160, 160, 64)    18496       tf_conv[0][0]
__________________________________________________________________________________________________
tf_c3 (TFC3)                    (1, 160, 160, 64)    18624       tf_conv_1[0][0]
__________________________________________________________________________________________________
tf_conv_7 (TFConv)              (1, 80, 80, 128)     73856       tf_c3[0][0]
__________________________________________________________________________________________________
tf_c3_1 (TFC3)                  (1, 80, 80, 128)     115200      tf_conv_7[0][0]
__________________________________________________________________________________________________
tf_conv_15 (TFConv)             (1, 40, 40, 256)     295168      tf_c3_1[0][0]
__________________________________________________________________________________________________
tf_c3_2 (TFC3)                  (1, 40, 40, 256)     623872      tf_conv_15[0][0]
__________________________________________________________________________________________________
tf_conv_25 (TFConv)             (1, 20, 20, 512)     1180160     tf_c3_2[0][0]
__________________________________________________________________________________________________
tf_c3_3 (TFC3)                  (1, 20, 20, 512)     1181184     tf_conv_25[0][0]
__________________________________________________________________________________________________
tfsppf (TFSPPF)                 (1, 20, 20, 512)     656128      tf_c3_3[0][0]
__________________________________________________________________________________________________
tf_conv_33 (TFConv)             (1, 20, 20, 256)     131328      tfsppf[0][0]
__________________________________________________________________________________________________
tf_upsample (TFUpsample)        (1, 40, 40, 256)     0           tf_conv_33[0][0]
__________________________________________________________________________________________________
tf_concat (TFConcat)            (1, 40, 40, 512)     0           tf_upsample[0][0]
                                                                 tf_c3_2[0][0]
__________________________________________________________________________________________________
tf_c3_4 (TFC3)                  (1, 40, 40, 256)     361216      tf_concat[0][0]
__________________________________________________________________________________________________
tf_conv_39 (TFConv)             (1, 40, 40, 128)     32896       tf_c3_4[0][0]
__________________________________________________________________________________________________
tf_upsample_1 (TFUpsample)      (1, 80, 80, 128)     0           tf_conv_39[0][0]
__________________________________________________________________________________________________
tf_concat_1 (TFConcat)          (1, 80, 80, 256)     0           tf_upsample_1[0][0]
                                                                 tf_c3_1[0][0]
__________________________________________________________________________________________________
tf_c3_5 (TFC3)                  (1, 80, 80, 128)     90496       tf_concat_1[0][0]
__________________________________________________________________________________________________
tf_conv_45 (TFConv)             (1, 40, 40, 128)     147584      tf_c3_5[0][0]
__________________________________________________________________________________________________
tf_concat_2 (TFConcat)          (1, 40, 40, 256)     0           tf_conv_45[0][0]
                                                                 tf_conv_39[0][0]
__________________________________________________________________________________________________
tf_c3_6 (TFC3)                  (1, 40, 40, 256)     295680      tf_concat_2[0][0]
__________________________________________________________________________________________________
tf_conv_51 (TFConv)             (1, 20, 20, 256)     590080      tf_c3_6[0][0]
__________________________________________________________________________________________________
tf_concat_3 (TFConcat)          (1, 20, 20, 512)     0           tf_conv_51[0][0]
                                                                 tf_conv_33[0][0]
__________________________________________________________________________________________________
tf_c3_7 (TFC3)                  (1, 20, 20, 512)     1181184     tf_concat_3[0][0]
__________________________________________________________________________________________________
tf_detect (TFDetect)            ((1, 25200, 85), [(1 229245      tf_c3_5[0][0]
                                                                 tf_c3_6[0][0]
                                                                 tf_c3_7[0][0]
__________________________________________________________________________________________________
tf.__operators__.getitem (Slici (1, 25200, 4)        0           tf_detect[0][0]
__________________________________________________________________________________________________
tf.split (TFOpLambda)           [(1, 25200, 1), (1,  0           tf.__operators__.getitem[0][0]
__________________________________________________________________________________________________
tf.math.truediv (TFOpLambda)    (1, 25200, 1)        0           tf.split[0][2]
__________________________________________________________________________________________________
tf.math.truediv_1 (TFOpLambda)  (1, 25200, 1)        0           tf.split[0][3]
__________________________________________________________________________________________________
tf.math.truediv_2 (TFOpLambda)  (1, 25200, 1)        0           tf.split[0][2]
__________________________________________________________________________________________________
tf.math.truediv_3 (TFOpLambda)  (1, 25200, 1)        0           tf.split[0][3]
__________________________________________________________________________________________________
tf.math.subtract (TFOpLambda)   (1, 25200, 1)        0           tf.split[0][0]
                                                                 tf.math.truediv[0][0]
__________________________________________________________________________________________________
tf.math.subtract_1 (TFOpLambda) (1, 25200, 1)        0           tf.split[0][1]
                                                                 tf.math.truediv_1[0][0]
__________________________________________________________________________________________________
tf.__operators__.add (TFOpLambd (1, 25200, 1)        0           tf.split[0][0]
                                                                 tf.math.truediv_2[0][0]
__________________________________________________________________________________________________
tf.__operators__.add_1 (TFOpLam (1, 25200, 1)        0           tf.split[0][1]
                                                                 tf.math.truediv_3[0][0]
__________________________________________________________________________________________________
tf.__operators__.getitem_2 (Sli (1, 25200, 80)       0           tf_detect[0][0]
__________________________________________________________________________________________________
tf.__operators__.getitem_1 (Sli (1, 25200, 1)        0           tf_detect[0][0]
__________________________________________________________________________________________________
tf.concat (TFOpLambda)          (1, 25200, 4)        0           tf.math.subtract[0][0]
                                                                 tf.math.subtract_1[0][0]
                                                                 tf.__operators__.add[0][0]
                                                                 tf.__operators__.add_1[0][0]
__________________________________________________________________________________________________
tf.math.multiply (TFOpLambda)   (1, 25200, 80)       0           tf.__operators__.getitem_1[0][0]
                                                                 tf.__operators__.getitem_2[0][0]
__________________________________________________________________________________________________
agnostic_nms_1 (AgnosticNMS)    ((1, None, 4), (1, N 0           tf.concat[0][0]
                                                                 tf.__operators__.getitem_2[0][0]
                                                                 tf.math.multiply[0][0]
==================================================================================================
Total params: 7,225,885
Trainable params: 0
Non-trainable params: 7,225,885
__________________________________________________________________________________________________
Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.
2021-11-01 23:22:50.000067: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
Found untraced functions such as tf_conv_2_layer_call_and_return_conditional_losses, tf_conv_2_layer_call_fn, tf_conv_3_layer_call_and_return_conditional_losses, tf_conv_3_layer_call_fn, tf_conv_4_laye
r_call_and_return_conditional_losses while saving (showing 5 of 520). These functions will not be directly callable after loading.
Assets written to: weights\yolov5s_saved_model\assets
TensorFlow saved_model: export success, saved as weights\yolov5s_saved_model (243.9 MB)

TensorFlow GraphDef: starting export with tensorflow 2.6.0...
2021-11-01 23:23:36.509836: I tensorflow/core/grappler/devices.cc:66] Number of eligible GPUs (core count >= 8, compute capability >= 0.0): 0
2021-11-01 23:23:36.520088: I tensorflow/core/grappler/clusters/single_machine.cc:357] Starting new session
2021-11-01 23:23:36.587873: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:1137] Optimization results for grappler item: graph_to_optimize
  function_optimizer: Graph size after: 1174 nodes (169), 1571 edges (274), time = 25.846ms.
  function_optimizer: function_optimizer did nothing. time = 1.137ms.

TensorFlow GraphDef: export success, saved as weights\yolov5s.pb (29.2 MB)

TensorFlow.js: export failure: module 'tensorflow.tools.docs.doc_controls' has no attribute 'inheritable_header'

Export complete (67.56s)
Results saved to F:\code\py\yolov5\weights
Visualize with https://netron.app

**Comment by glenn-jocher** [comment]
@gamefake588 closing as duplicate of https://github.com/ultralytics/yolov5/issues/5434
